core/input/input.cpp

```diff
@@ -59,21 +59,21 @@ static const char *_joy_buttons[JOY_SDL_BUTTONS + 1] = {
 };
 
 static const char *_joy_button_names[JOY_BUTTON_MAX] = {
-	"Face Bottom",
-	"Face Right",
-	"Face Left",
-	"Face Top",
+	"Bottom Action",
+	"Right Action",
+	"Left Action",
+	"Top Action",
```

Comment:
I don't like this change at all. Why "Top Action" but not "Select Action" or "Guide Action"? What is "Top" the top of, if we're not specifying that these are the face buttons?

I'm also wondering if it would be a better idea to just go with "A button", "B button", etc. Pretty much every controller except for Sony and Nintendo controllers use the SDL layout (based on the Xbox layout), including Xbox controllers, Stadia controller, Steam controllers, most Android controllers, and most other generic controllers.